.github/workflows/deploy-prod.yml

```
@@ -61,6 +85,10 @@ jobs:
 
   deploy-proxy:
     if: ${{ inputs.deploy-proxy }}
+    env:
```

actionlint (1.7.8)

when a reusable workflow is called with "uses", "env" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "deploy-proxy"

(syntax-check)